__tests__/add-to-project.test.ts

```diff
@@ -179,6 +179,209 @@ describe('addToProject', () => {
     expect(infoSpy).toHaveBeenCalledWith(`Skipping issue 1 because it doesn't match all the labels: bug, new`)
     expect(gqlMock).not.toHaveBeenCalled()
   })
+
+  test('adds matching issues with multiple label filters', async () => {
+    mockGetInput({
+      'project-url': 'https://github.com/orgs/github/projects/1',
+      'github-token': 'gh_token',
+      labeled: 'accessibility,backend,bug'
+    })
+
+    github.context.payload = {
+      issue: {
+        number: 1,
+        labels: [{name: 'accessibility'}, {name: 'backend'}]
+      }
+    }
+
+    const gqlMock = mockGraphQL(
+      {
+        test: /getProject/,
+        return: {
+          organization: {
+            projectNext: {
+              id: 'project-next-id'
+            }
+          }
+        }
+      },
+      {
+        test: /addProjectNextItem/,
+        return: {
+          addProjectNextItem: {
+            projectNextItem: {
+              id: 'project-next-item-id'
+            }
+          }
+        }
+      }
+    )
+
+    const infoSpy = jest.spyOn(core, 'info')
+
+    await addToProject()
+
+    expect(gqlMock).toHaveBeenCalled()
+    expect(infoSpy).not.toHaveBeenCalled()
+    expect(outputs.itemId).toEqual('project-next-item-id')
+  })
+
+  test('does not add un-matching issues with multiple label filters', async () => {
+    mockGetInput({
+      'project-url': 'https://github.com/orgs/github/projects/1',
+      'github-token': 'gh_token',
+      labeled: 'accessibility,backend,bug'
```

Comment:
what happens if the label has a `,` in it? can we support that?

do we need to consider other syntax to support that?  What about other characters, spaces, etc?

Reply:
Luckily labels can't be created with `,` but good point nonetheless about spaces and other edge cases 👍

Reply:
Bit off-topic to this PR, but would be great to pass off all of these concerns to a separate library just for filtering (e.g., https://github.com/github/memex-filter-parser) and have a single `filter:` input here.

Reply:
@camchenry I'm getting a `404`

Reply:
@camchenry eventually that might make sense, but for this specific implementation, we only support a handful of fields, `label` `assignee` `milestone` (and I think that's probably it)

extracting the parser could be fun though